### DISCUSSION TITLE
Full Text Search in binary datatype in lancedb

### DISCUSSION THREAD

**Discussion by Raghulkannan14** [lancedb/lancedb#1237] [Q&A]
full_name: binary I am having the full_name as binary data type I need to do a full text search on that column if i create a index its throws a error TypeError: Field full_name is not a string type how to create a index for binary data type and to do search in lancedb

**Comment by chebbyChefNEQ** [comment]
Hi,

Since the data is of binary type, it is not possible to create an FTS index on it. We need to know that the data is utf-8 encoded before we can create the FTS index.
